src/EditorFeatures/CSharpTest/ImplementAbstractClass/ImplementAbstractClassTests.cs

```diff
@@ -1818,6 +1818,171 @@ public override void M<T>()
     {
         throw new System.NotImplementedException();
     }
+}");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsImplementAbstractClass)]
```

Comment:
should likely have WorkItem attributes.

Reply:
Add `[WorkItem(39256, ...)]` for every test added in this PR?

Reply:
yup!